v2/pkg/engine/resolve/datasource.go

```diff
@@ -13,7 +13,6 @@ type DataSource interface {
 	Load(ctx context.Context, input []byte, out *bytes.Buffer) (err error)
 	LoadWithFiles(ctx context.Context, input []byte, files []*httpclient.FileUpload, out *bytes.Buffer) (err error)
 }
```

Comment:
unnecessary change

Reply:
I initially had another function for a datasource in there but we decided that we want to postpone having a new interface with variables as a typed argument in the future, I will revert it